component/otelcol/internal/featuregate/install_test.go

```diff
@@ -1,13 +0,0 @@
-package featuregate
-
-import (
-	"testing"
-
-	"github.com/stretchr/testify/require"
-	"go.opentelemetry.io/collector/featuregate"
-)
-
-func Test_enableFeatureGates(t *testing.T) {
-	err := enableFeatureGates(featuregate.GlobalRegistry())
-	require.NoError(t, err, "enableFeatureGates should not have a failed. Did a feature gate get removed from upstream?")
-}
```

Comment:
nit: it's unfortunate to remove this test. We should bring it back.

Reply:
Ok sure, I added it in a slightly different way so that it tests more. I hope you're ok with it?